Meadowlark-js/backends/meadowlark-opensearch-backend/test/integration/QueryOpensearch.test.ts

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import {
+  ResourceInfo,
+  UpsertRequest,
+  NoDocumentInfo,
+  newSecurity,
+  UpsertResult,
+  DeleteRequest,
+  DeleteResult,
+  PaginationParameters,
+  QueryRequest,
+} from '@edfi/meadowlark-core';
+import { Client } from '@opensearch-project/opensearch/.';
+import { getNewClient } from '../../src/repository/Db';
+import { queryDocuments } from '../../src/repository/QueryOpensearch';
+import { afterUpsertDocument, afterDeleteDocumentById } from '../../src/repository/UpdateOpensearch';
+import { setupOpenSearch, teardownOpenSearch } from '../setup/OpenSearchSetup';
+
+jest.setTimeout(120_000);
```

Comment:
An underscore?

Reply:
Yes, this is just "syntactic sugar" to express a bigger number. I left it the same as the value on the setup open search, but we could decrease it if needed, it does need a bigger value than the current default of 40000, but we could try and see to avoid intermittent errors

Reply:
But what does it mean? Having a longer time is fine, but I have no clue how to interpret that 😆 .

Reply:
Got it. It's 120000